frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield.jsx

```diff
@@ -99,12 +95,6 @@ const ErrorMessage = ({ error }) => {
 export default class ExpressionEditorTextfield extends React.Component {
   constructor() {
     super();
-    // memoize processSource for performance when editing previously seen source/targetOffset
-    this._processSource = memoize(processSource, ({ source, targetOffset }) =>
```

Comment:
This memoization is not necessarily anymore since we're not calling this function frequently.